static/pdf-preview-styles.css

```diff
@@ -10,6 +9,10 @@
   }
 }
 
+img {
```

Comment:
this looks a bit dangerous? what about images that have a lower intrinsic height? They'd maybe get stretched? And I assume this is trying to solve the problem of not knowing image sizes? Can we instead figure out firing a callback when all images have loaded or so?

Not a blocker! But let's chat about this one.

Reply:
@batpad this file is not currently being used. It was previously used by the pagedjs but not anymore. I'll clean-up this file and other pagedjs related stuff once we are sure about this method

Reply:
Ah, I see this is not actually being used any more 👍